app/Mail/NewRegistration.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class NewRegistration extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    public string $recipient;
+    public string $registered_user;
+    public bool $is_collegist;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct(string $recipient, string $registered_user, bool $is_collegist)
+    {
+        $this->recipient = $recipient;
+        $this->registered_user = $registered_user;
+        $this->is_collegist = $is_collegist;
```

Comment:
Here passing only $user is enough. This way you can use the user's other attributes later as well.